.prettierrc

```diff
@@ -1,3 +0,0 @@
-{
-  "bracketSpacing": true
-}
```

Comment:
Why'd we update this? Looks like it caused a bunch of changes in `cSpell.json`.

Reply:
had moved config to `package.json` for prettier for reasons that idt are relevant anymore, will restore this